src/main/webapp/game.html

```diff
@@ -24,14 +24,15 @@
             <div class="col s12">
               <p 
               id="stat"><p>
-              <!-- <hr width="80%"> -->
             </div>
           </div>
           
 
 
           <!-- Prompt -->
-          <div id="prompt-container" class="row center-align valign-wrapper">
+          <div 
+            id="prompt-container" 
```

Comment:
nit: "When line-wrapping, each continuation line should be indented at least 4 additional spaces from the original line" - google style guide. 2 spaces honestly seems fine to me though.

Reply:
Thanks for pointing this out. I just instinctively indented 2 spaces. I will fix it in a future commit.